docker-compose.yaml

```diff
@@ -40,15 +41,16 @@ services:
         target: /workspace/node_modules
     ports:
       - target: 80
+        published: 80
```

Comment:
Sort of. If you use `docker run runner` it doesn't publish any ports by default so there is no conflict. However if you run `docker up runner server` there will be a conflict.

Reply:
Aaaah, ok that makes sense. I noticed that the env vars also aren't present when running `docker run runner bash` which is unfortunate. TBH we could probably remove the ports _and_ env variables from the `runner` definition, since they are more confusing than anything else.